lib/app/views/dashboard.erb

```diff
@@ -15,13 +17,28 @@
           <% if current_user.ongoing.any? %>
             <p>Keep working on these problems, or mark them as done.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% current_user.ongoing.each do |submission| %>
+              <% current_user.ongoing[0..4].each do |submission| %>
```

Comment:
Could you explain how it's requesting all of them? I think I might be misunderstanding the code. Here is my thought process:
- `if current_user.ongoing.any?` checks if there are any (doesn't pull all of them)
- `current_user.ongoing[0..4].each do |submission|` only loops through the first 5 (also doesn't) pull all of them